samples/subsys/fs/littlefs/boards/stm32h747i_disco_m7.conf

```diff
@@ -0,0 +1,12 @@
+CONFIG_COMMON_LIBC_MALLOC_ARENA_SIZE=8192
+CONFIG_SDMMC_STM32=y
+CONFIG_SHELL=y
+CONFIG_FILE_SYSTEM_SHELL=y
+CONFIG_SDMMC_STM32_HWFC=y
```

Comment:
This is actually right and this config should be removed from prj_blk.conf (and added in nucleo_h743zi_blk.conf)